libs/core/langchain_core/language_models/chat_models.py

```diff
@@ -204,7 +216,7 @@ def stream(
         else:
             config = ensure_config(config)
             messages = self._convert_input(input).to_messages()
-            params = self._get_invocation_params(stop=stop, **kwargs)
+            params = self._get_ls_params(stop=stop, **kwargs)
```

Comment:
@nfcampos I am setting `invocation_params=params` in `callback_manager.on_chat_model_start`. is that the correct thing to do?